Gems/ScriptCanvas/Code/Editor/View/Windows/ScriptCanvasContextMenus.cpp

```diff
@@ -805,6 +810,13 @@ namespace ScriptCanvasEditor
     SceneContextMenu::SceneContextMenu(const NodePaletteModel& paletteModel, AzToolsFramework::AssetBrowser::AssetBrowserFilterModel* assetModel)
         : GraphCanvas::SceneContextMenu(ScriptCanvasEditor::AssetEditorId)
     {
+
+        auto userSettings = AZ::UserSettings::CreateFind<EditorSettings::ScriptCanvasEditorSettings>(AZ_CRC("ScriptCanvasPreviewSettings", 0x1c5a2965), AZ::UserSettings::CT_LOCAL);
+        if (userSettings)
+        {
+            m_userNodePaletteWidth = userSettings->m_sceneContextMenuNodePaletteWidth;
+        }
```

Comment:
You should prefer to use the [SettingsRegistry](https://github.com/o3de/o3de/blob/development/Code/Editor/Settings.cpp#L1104-L1147) instead of the UserSettings

Reply:
That would imply refactoring the entire Script Canvas editor settings as opposed to adding a new setting to the existing system. While I do agree with you, it's out of scope at the moment.

Reply:
I am good with your current change then, since you are using the existing UserSettings that is already there.